From d00885265d21017c4ad8cdd4a52aa3b1637e9f7f Mon Sep 17 00:00:00 2001
From: Andreas Metzler <ametzler@bebt.de>
Date: Sat, 4 Sep 2021 14:07:48 +0200
Subject: [PATCH] build: Use automake primitives to install libksba.def
Forwarded: https://dev.gnupg.org/T5579

* src/Makekefile.am: Do not use explicit $INSTALL) to install
  libksba.def, fixes windows parallel build error.
---
 src/Makefile.am | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 0eedae7..b3343ca 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -65,13 +65,12 @@ no_undefined = -no-undefined
 export_symbols = -export-symbols $(srcdir)/libksba.def
 extra_ltoptions = -XCClinker -static-libgcc
 
-install-def-file:
-	$(INSTALL) $(srcdir)/libksba.def $(DESTDIR)$(libdir)/libksba.def
-
 uninstall-def-file:
 	-rm $(DESTDIR)$(libdir)/libksba.def
 
 ksba_deps = $(ksba_res) libksba.def
+ksbadefdir = $(libdir)
+ksbadef_DATA = libksba.def
 
 else !HAVE_W32_SYSTEM
 ksba_res =
@@ -79,7 +78,6 @@ no_undefined =
 export_symbols =
 extra_ltoptions =
 ksba_deps =
-install-def-file:
 uninstall-def-file:
 endif !HAVE_W32_SYSTEM
 
@@ -137,6 +135,4 @@ asn1-tables.c : $(asn1_modules) asn1-gentables
 	  ./asn1-gentables $$list > asn1-tables.c
 
 
-install-data-local: install-def-file
-
 uninstall-local: uninstall-def-file
-- 
2.33.0

